jbuilder.1.0+beta1 checksum

The following checksums doesn't match! It's a fresh opam install.

```
The following actions would be performed:
  ∗  install   bisect      1.3                    
  ↘  downgrade jbuilder    1.0+beta17 to 1.0+beta1
  ∗  install   base-bytes  base                   
  ∗  install   rmtld3synth 0.3-alpha*             
  ∗  install   ounit       2.0.7                  
  ∗  install   qcheck      0.8                    
  ∗  install   qtest       2.7                    
===== ∗  6   ↘  1 =====
# Detecting depexts using flags: x86_64 linux ubuntu
# The following system packages are needed:
#  - debianutils
#  - m4
# All required OS packages found.
opam install --unset-root "qtest.2.7" "qcheck.0.8" "jbuilder.1.0+beta1" "ounit.2.0.7" "base-bytes.base" "bisect.1.3"
The following actions will be performed:
  ∗  install   bisect     1.3                    
  ↘  downgrade jbuilder   1.0+beta17 to 1.0+beta1
  ∗  install   base-bytes base                   
  ∗  install   ounit      2.0.7                  
  ∗  install   qcheck     0.8                    
  ∗  install   qtest      2.7                    
===== ∗  5   ↘  1 =====
=-=- Gathering sources =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
[ERROR] Bad checksum for
        /home/travis/.opam/packages.dev/jbuilder.1.0+beta1/1.0+beta1.tar.gz:
          - 4cc5b4b2ab5b9165e1664a9155e820fd [expected result]
          - 239eee4a2c9e867551c74314c41369d7 [actual result]
        This may be fixed by running `opam update`.
[qcheck.0.8] https://github.com/c-cube/qcheck/archive/0.8.tar.gz downloaded
[ounit.2.0.7] https://forge.ocamlcore.org/frs/download.php/1738/ounit-2.0.7.tar.gz downloaded
[qtest.2.7] https://github.com/vincent-hugot/iTeML/archive/v2.7.tar.gz downloaded
[bisect.1.3] http://bisect.x9c.fr/distrib/bisect-1.3.tar.gz downloaded
[ERROR] The sources of the following couldn't be obtained, aborting:
          - jbuilder.1.0+beta1
        (This may be fixed by running 'opam update')
'unset TESTS; OPAMYES=1 export OPAMYES; unset OPAMBUILDTEST; set -ue; opam install --unset-root "qtest.2.7" "qcheck.0.8" "jbuilder.1.0+beta1" "ounit.2.0.7" "base-bytes.base" "bisect.1.3"' exited 66. Terminating with 66
```

--- a/packages/jbuilder/jbuilder.1.0+beta1/url
+++ b/packages/jbuilder/jbuilder.1.0+beta1/url
@@ -1,2 +1,2 @@
 archive: "https://github.com/janestreet/jbuilder/archive/1.0+beta1.tar.gz"
-checksum: "4cc5b4b2ab5b9165e1664a9155e820fd"
+checksum: "239eee4a2c9e867551c74314c41369d7"

